validation/urls.py

```diff
@@ -51,4 +51,5 @@
         views.all_recordings_for_session,
         name="crude_recordings",
     ),
+    path("<str:segment_id>", views.segment_content_view, name="segment_detail"),
```

Comment:
```suggestion
    path("segment/<str:segment_id>", views.segment_content_view, name="segment_detail"),
```
Otherwise the log fills up with error messages about `/favicon.ico` not parsing as an integer